src/content/basics/nodepools/index.md

```diff
@@ -189,6 +163,17 @@ In case there are workloads not assigned to any node pools, the autoscaler may p
 
 ## Limitations
 
+- Clusters without worker nodes (= without node pools) cannot be considered fully functional. In order to have all
+required components scheduled, worker nodes are required. For that reason, we deactivate any monitoring and alerts for
+these clusters and don't provide any proactive support.
+
+- We also do not monitor node pools with less than three worker nodes and do not provide any proactive support for those.
+
+- When creating a new node pool, the master node of the cluster is re-created. This causes a downtime of the Kubernetes
```

Comment:
Is this true??? Why???

Reply:
I don't know, it's different on Azure? I think the three nodes node pool is a bit extreme, we have a limit of at least 3 nodes per cluster but we still monitor and support it afaik. Should we ask Firecracker?

Reply:
Are we recrating the master node if we create a NodePool?

Reply:
this was true in the old AWS architecture, now with ASG masters we do not recreate master